main.go

```diff
@@ -32,7 +33,7 @@ import (
 )
 
 func main() {
-	fmt.Printf("\nCrossdock starting...\n\n")
+	fmt.Printf("\nCrossdock starting (go %v)...\n\n", runtime.Version())
```

Comment:
I'm not sure if this belongs on the startup print. Of all the information in the crossdock environment that might be useful, I'm not sure the Go version is the most important.

While you were debugging your specific issue, I'm sure this came in handy, but do you expect many problems where this will come in useful?